portuguese-pt.ini

```
@@ -0,0 +1,661 @@
+Left_Tip = "Muda o botão designado para Esquerda"
+Right_Tip = "Muda o botão designado para Direita"
+Jump_Tip = "Muda o botão designado para Saltar"
+Fire_Tip = "Muda o botão designado para Disparar"
+DiagonalAim_Tip = "Muda o botão designado para Apontar Diagonalmente"
+AimUp_Tip = "Muda o botão designado para Apontar para Cima"
+AimDown_Tip = "Muda o botão designado para Apontar para Baixo"
+AimLock_Tip = "Muda o botão designado para Fixar Mira"
+Morph_Tip = "Muda o botão designado para Morph"
+ArmWeapon_Tip = "Muda o botão designado para Activar Mísseis"
+WeaponSelect_Tip = "Muda o botão designado para Seleccionar Arma"
+WeaponCancel_Tip = "Muda o botão designado para Cancelar Selecção de Arma"
+Start_Tip = "Muda o botão designado para Começar / Menu"
+MenuAccept_Tip = "Muda o botão utilizado para aceitar opções nos menus"
+MenuBack_Tip = "Muda o botão utilizado para voltar ao menu anterior"
+
+[PauseMenu]
+Resume = "Continuar Jogo"
+Resume_Tip = "Continuar com o jogo actual"
+Restart = "Recomeçar do último Jogo Gravado"
+Restart_Tip = "Abandonar o jogo actual e recomeçar do último gravado"
+Options = "Opções"
+Options_Tip = "Mudar as definições de jogo"
+Quit = "Sair do Jogo"
+Quit_Tip = "Abandonar o jogo actual e sair para o Menu Principal"
+
+[QuitMenu]
+Exit = "Sair - Tem a certeza?"
+Reload = "Recarregar - Tem a certeza?"
+Warning = "ATENÇÃO: Todo o progresso não guardado será perdido"
+
+[Subscreen]
+Title_Map = "PLANETA SR-388"
+Title_Options = "Menu Pausa"
+Menu_Tip = "| -> Menu"
+Marker_Tip = "| - Colocar Marcador"
+MarkerPos_Tip = "| - Definir Posição do Marcador"
+DeleteMarker_Tip = "| - Apagar Marcador"
+EditMarker_Tip = "| - Editar Marcador"
+SelectItem_Tip = " - Seleccionar item"
+ToggleItem_Tip = "| - Activar item"
+Title_Logs = "Registo de Missão"
+SelectCategory_Tip = " - Seleccionar Categoria"
+SelectLog_Tip = " - Seleccionar Registo"
+ExpandLog_Tip = "| - Expandir Registo"
+ScrollText_Tip = " - Mover Texto"
+Back_Tip = "| - Voltar atrás"
+
+[Equipment]
+PowerSuit_Tip = "Fato Blindado Padrão de Samus."
+VariaSuit_Tip = "Oferece protecção adicional contra inimigos e ambientes perigosos."
+GravitySuit_Tip = "Protege contra todos os perigos, impede fricção quando submergido."
+ChargeBeam_Tip = "Manter pressionado | para concentrar energia, largar para disparar."
+IceBeam_Tip = "Congela inimigos instantaneamente. Dano adicional."
+WaveBeam_Tip = "Os raios penetram objectos sólidos."
+SpazerBeam_Tip = "Largura dos raios aumentada."
+PlasmaBeam_Tip = "Os raios penetram inimigos."
+MorphBall_Tip = "Samus transforma-se numa bola e passa por espaços estreitos."
+SpiderBall_Tip = "Samus pode colar-se a paredes e tectos enquanto Morph Ball."
+JumpBall_Tip= "Permite Samus saltar enquanto Morph Ball."
+Bombs_Tip = "Permite colocar bombas enquanto Morph Ball."
+PowerGrip_Tip = "Samus pode agarrar-se a saliências e puxar-se para cima."
+ScrewAttack_Tip = "Despoleta uma quantidade de energia letal enquanto usa o Space Jump."

```

Space Jump should be Spin Jump.
```suggestion
ScrewAttack_Tip = "Despoleta uma quantidade de energia letal enquanto usa o Spin Jump."

```

---

Hm, not sure with how anglicisms are used in Portuguese, but maybe replace `Spin Jump` with whatever the portuguese translation of `(to) summersault` is?

---

Fixed. Changed to Spin Jump.